test/Driver/lines_to_execute.swift

```diff
@@ -0,0 +1,26 @@
+// RUN: %swift_driver_plain -e 'print("Hello, world!")' 2>&1 | %FileCheck -check-prefix CHECK-1 %s
```

Comment:
These should just use `%swift_driver` rather than `%swift_driver_plain` so that they still get the right target and module cache. You'll also need `REQUIRES: swift_interpreter` so that they don't fail on non-host tests or platforms where the JIT isn't set up.